Cargo.toml

```diff
@@ -19,14 +19,11 @@ rustdoc-args = ["--generate-link-to-definition"]
 [features]
 default = [
     "device_arcade_button_1x4",
-    "device_neokey_1x4",
-    "device_neorotary4",
-    "device_neoslider",
-    "device_neotrellis",
-    "device_rotary_encoder",
```

Comment:
i think it's fine to keep devices in the default list if their main functionality is supported without nightly - e.g. the rotary encoder works just fine (AFAIK just the neopixel module needs nightly?)